src/app/productentemplate-overzicht/add-product-template-form/add-product-template-form.component.ts

```diff
@@ -16,8 +16,8 @@ import { CategoryTemplate } from 'src/app/types/categoryTemplate.model';
 })
 export class AddProductTemplateFormComponent implements OnInit {
 
-  public error: string = '../../assets/images/error.svg';
-  public correct: string = '../../assets/images/correct.svg';
+  public error = '../../assets/images/error.svg';
+  public correct = '../../assets/images/correct.svg';
```

Comment:
Het is beter om te linken naar afbeeldingen in de assets zonder de '..' dit kan problemen geven bij de build omdat de code die je nu schrijft dan niet meer op dezelfde plaats zal staan waardoor de relatieve paden niet meer werken.
Hier zou dit dus moeten zijn 'assets/images/error.svg' en dit zou moeten werken

Reply:
aangepast